### PR TITLE
fix(@wallet): Fix text area for importing private key

### LINKED ISSUE #4535: [base_bc] can't import private key in wallet
### Description

Screenshot 2022-01-24 at 14 25 05

### PR DESCRIPTION
**Pull request by alaibe**
fixes #4535

### CODE REVIEW

File: ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml [hunk 1]
```diff
@@ -81,17 +81,28 @@ ModalPopup {
...
-
     StyledTextArea {
         id: accountPKeyInput
+        customHeight: 88
         anchors.top: passwordInput.bottom
         anchors.topMargin: marginBetweenInputs
+        anchors.left: parent.left
+        anchors.right: parent.right
+        validationError: popup.privateKeyValidationError
+        //% "Private key"
```
Comment (jrainville) [inline ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml:92]: Can remove this, or maybe the label needs to be added back?

Comment (alaibe) [reply ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml:92]: Third option, push :facepalm: label added back. thanks

**Review by jrainville — APPROVED** [review]

File: ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml [hunk 2]
```diff
@@ -81,17 +81,29 @@ ModalPopup {
...
+        anchors.right: parent.right
+        validationError: popup.privateKeyValidationError
         //% "Private key"
         label: qsTrId("private-key")
-        customHeight: 88
-        validationError: popup.privateKeyValidationError
+        textField.wrapMode: Text.WordWrap
+        textField.horizontalAlignment: TextEdit.AlignHCenter
+        textField.verticalAlignment: TextEdit.AlignVCenter
+        textField.font.pixelSize: 15
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml:97]: this is already set as default inside the StyledTextArea.

File: ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml [hunk 3]
```diff
@@ -81,17 +81,29 @@ ModalPopup {
...
+        textField.font.weight: Font.DemiBold
+        //% "Paste the contents of your private key"
+        placeholderText: qsTrId("paste-the-contents-of-your-private-key")
+        textField.placeholderTextColor: Style.current.secondaryText
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml:101]: same as above

File: ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml [hunk 4]
```diff
@@ -81,17 +81,29 @@ ModalPopup {
...
+        textField.selectByMouse: true
+        textField.selectByKeyboard: true
+        textField.selectionColor: Style.current.secondaryBackground
+        textField.selectedTextColor: Style.current.secondaryText
+        textField.color: Style.current.textColor
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml:106]: some of these are already default values in the StyledTextArea, can you only change the ones that need customisation

**Review by Khushboo-dev-cpp — APPROVED** [review]

File: ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml (hunk 4 above)
Comment (alaibe) [reply ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml]: Updated! good catch

**Review by iurimatias — APPROVED** [review]

**Comment by iurimatias** [comment]
tested

**Pull request merged**